pom.xml

```diff
@@ -6,7 +6,12 @@
 
     <groupId>io.xpring</groupId>
     <artifactId>xpring4j</artifactId>
-    <version>1.0-SNAPSHOT</version>
+    <version>1.0.0</version>
```

Comment:
The `master` branch should never have a release version in it. Instead, checkout a tag for `1.0.0` and release from there. The `master` branch should _always_ be a `SNAPSHOT`.